libs/analytics/README.md

```diff
@@ -1,13 +1,3 @@
-# WordPressCom-Analytics-Android
+# Analytics
 
-Library for handling Analytics tracking in WordPress Android.
-
-Part of the [WordPress-Android] project.
-
-[WordPress-Android]: https://github.com/wordpress-mobile/WordPress-Android
-
-## Publish it to bintray
-
-```shell
-$ ./gradlew assemble publishToMavenLocal bintrayUpload -PbintrayUser=FIXME -PbintrayKey=FIXME -PdryRun=false
-```
+Library for handling Analytics tracking.
```

Comment:
This is not a library anymore. I vote for removing the `README` alltogether, but if we are keeping, I think we should call it a `module` instead.

Reply:
👋 @oguzkocer and thanks!

I was thinking the same, I was in favor of removing this `README.md` file altogether too. The only reason that I kept that there is in case we might want to have a `README.md`, file per module, just as a general rule, so that to allow engineers to contribute to it by describing what the module does, why it is needed or any other FAQ.

> ...but if we are keeping, I think we should call it a module instead.

💯 Let me know what you think of the above and if you still think we should remove the `README.md` anyway I'll just remove it. Else, I'll just rename `Library` to `Module`, good call. 👍

Reply:
I don't feel strongly about having a `README` or not. Now that you renamed `Library` to `Module`, it's all good from my side 👍

Reply:
Great, thank you @oguzkocer ! 🙇 

PS: I'll keep the basic structure of the `README` for now (`title` + `short description`), add that to every module as I will be progressing with this restructuring, that is, if such file is missing, and maybe come back to it later by adding some more details on them, most probably with some additional help from WP engineers!